src/main/java/org/ibp/api/java/impl/middleware/study/StudyEntryMapper.java

```diff
@@ -78,4 +80,16 @@ protected void configure() {
 			}
 		});
 	}
+
+	private static void addGermplasmMapping(final ModelMapper mapper) {
+		mapper.addMappings(new PropertyMap<Germplasm, StudyEntryDto>() {
+
+			@Override
+			protected void configure() {
+				this.map().setGid(this.source.getGid());
+				this.map().setDesignation(this.source.getPreferredName().getNval());
+				//this.map().setEntryCode(this.source.getEntryCode());
```

Comment:
What should be the value of entry code?

Reply:
The entry code's value will be the same as the entry number. It will be set in the code where entry number is being set. I'll just remove this commented out code.